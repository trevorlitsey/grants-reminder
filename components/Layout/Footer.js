const Footer = () => (
	<div>
		<p>reminder emails are sent daily at 8am</p>
		<style jsx>{`

			div {
				margin: 28px;
			}

			p {
				color: HSLA(225, 9%, 74%, 1.00);
				text-align: center;
			}
		`}</style>
	</div>
)

export default Footer;