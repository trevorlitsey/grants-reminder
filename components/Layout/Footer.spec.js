import React from 'react';
import { shallow } from 'enzyme';

import Footer from './Footer';

it('Footer should render without crashing', () => {
	const wrapper = shallow(<Footer />)
	expect(wrapper).toMatchSnapshot();
})