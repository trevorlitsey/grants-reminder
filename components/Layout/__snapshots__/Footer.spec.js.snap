// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <p
          className="jsx-853876180"
        >
          reminder emails are sent daily at 8am
        </p>,
        <JSXStyle
          css="div.jsx-853876180{margin:28px;}p.jsx-853876180{color:HSLA(225,9%,74%,1.00);text-align:center;}"
          styleId="853876180"
        />,
      ],
      "className": "jsx-853876180",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "reminder emails are sent daily at 8am",
          "className": "jsx-853876180",
        },
        "ref": null,
        "rendered": "reminder emails are sent daily at 8am",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "css": "div.jsx-853876180{margin:28px;}p.jsx-853876180{color:HSLA(225,9%,74%,1.00);text-align:center;}",
          "styleId": "853876180",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <p
            className="jsx-853876180"
          >
            reminder emails are sent daily at 8am
          </p>,
          <JSXStyle
            css="div.jsx-853876180{margin:28px;}p.jsx-853876180{color:HSLA(225,9%,74%,1.00);text-align:center;}"
            styleId="853876180"
          />,
        ],
        "className": "jsx-853876180",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "reminder emails are sent daily at 8am",
            "className": "jsx-853876180",
          },
          "ref": null,
          "rendered": "reminder emails are sent daily at 8am",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "css": "div.jsx-853876180{margin:28px;}p.jsx-853876180{color:HSLA(225,9%,74%,1.00);text-align:center;}",
            "styleId": "853876180",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
